api/v1beta1/azuremanagedcontrolplane_types.go

```diff
@@ -213,6 +213,10 @@ type AzureManagedControlPlaneSpec struct {
 	// OIDCIssuerProfile is the OIDC issuer profile of the Managed Cluster.
 	// +optional
 	OIDCIssuerProfile *OIDCIssuerProfile `json:"oidcIssuerProfile,omitempty"`
+
+	// DisableLocalAccounts - If set to true, getting static credential will be disabled for this cluster. Expected to only be used for AAD clusters.
+	// +optional
+	DisableLocalAccounts *bool `json:"disableLocalAccounts,omitempty"`
```

Comment:
Is there a difference between how we handle nil and false? I'm thinking we could just make it a normal bool to avoid unecessary pointers.

Reply:
In the entire spec all optional fields are generally pointers :)